sql/updates/world/master/2023_99_99_99_aberrus.sql

```diff
@@ -0,0 +1,448 @@
+(20970, 90307, @CGUID+75, 1, 0, 0, 0, 0, 50000), -- Full: 0x2042554120C5AC000069F50000666449 Creature/0 R4245/S27125 Map: 2569 (Aberrus, the Shadowed Crucible) Entry: 202416 (Scalecommander Sarkareth) Low: 6710345
+(20970, 89966, @CGUID+4, 0, 0, 0, 0, 0, 50000), -- Full: 0x2042554120C4D9C00069F50000666448 Creature/0 R4245/S27125 Map: 2569 (Aberrus, the Shadowed Crucible) Entry: 201575 (Sabellian) Low: 6710344
+(20985, 90307, @CGUID+75, 2, 0, 0, 0, 0, 50000), -- Full: 0x2042554120C5AC000069F50000666449 Creature/0 R4245/S27125 Map: 2569 (Aberrus, the Shadowed Crucible) Entry: 202416 (Scalecommander Sarkareth) Low: 6710345
+(20985, 90376, @CGUID+73, 0, 0, 0, 0, 0, 50000), -- Full: 0x2042554120C5DC800069F50000666448 Creature/0 R4245/S27125 Map: 2569 (Aberrus, the Shadowed Crucible) Entry: 202610 (Winglord Dezran) Low: 6710344
+(20985, 90395, @CGUID+70, 1, 0, 0, 0, 0, 50000); -- Full: 0x2042554120C5E3400069F50000666448 Creature/0 R4245/S27125 Map: 2569 (Aberrus, the Shadowed Crucible) Entry: 202637 (Zskarn) Low: 6710344
+
+DELETE FROM `conversation_line_template` WHERE `Id` IN (56693, 56692, 56691, 56690, 56694, 56695, 56696, 56697, 56698, 56699, 56700, 56701, 56702, 57836, 57837, 57838, 57821);
+INSERT INTO `conversation_line_template` (`Id`, `UiCameraID`, `ActorIdx`, `Flags`, `ChatType`, `VerifiedBuild`) VALUES
+(56693, 0, 0, 0, 0, 50000),
+(56692, 0, 0, 0, 0, 50000),
+(56691, 0, 1, 0, 0, 50000),
+(56690, 0, 0, 0, 0, 50000),
+(56696, 0, 1, 0, 0, 50000),
+(56695, 0, 1, 0, 0, 50000),
+(56694, 0, 0, 0, 0, 50000),
+(57821, 0, 2, 0, 0, 50000),
+(56702, 0, 0, 0, 0, 50000),
+(56701, 0, 2, 0, 0, 50000),
+(57838, 0, 1, 0, 0, 50000),
+(57837, 0, 1, 0, 0, 50000),
+(57836, 0, 2, 0, 0, 50000),
+(56700, 0, 1, 0, 0, 50000),
+(56699, 0, 1, 0, 0, 50000),
+(56698, 0, 1, 0, 0, 50000),
+(56697, 0, 0, 0, 0, 50000);
+
+DELETE FROM `conversation_template` WHERE `Id`IN (20800, 20985, 20970);
+INSERT INTO `conversation_template` (`Id`, `FirstLineId`, `TextureKitId`, `ScriptName`, `VerifiedBuild`) VALUES
+(20800, 56690, 0, 'conversation_aberrus_sabellian_intro', 50000),
+(20970, 56694, 0, '', 50000),
+(20985, 56697, 0, 'conversation_aberrus_kazzara_intro', 50000);
+
+-- Conditions
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=13 AND `SourceGroup`IN (1,2) AND `SourceEntry`IN (411873, 411877, 411874);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(13, 1, 411873, 0, 0, 31, 0, 4, 0, 0, 0, 0, 0, '', 'Spell 411873 can only hit Player'),
+(13, 2, 411873, 0, 0, 31, 0, 4, 0, 0, 0, 0, 0, '', 'Spell 411873 can only hit Player'),
+(13, 1, 411877, 0, 0, 31, 0, 3, 205734, 0, 0, 0, 0, '', 'Spell 411877 can only hit Sundered Edgelord'),
```

Comment:
you can group the conditions by defining them once for a reference and then just check for the reference in both cases (411877, 411874)